src/mngt/ocf_mngt_cache.c

```diff
@@ -2075,7 +2075,14 @@ void ocf_mngt_cache_load(ocf_cache_t cache,
 
 	/* Load is not allowed in volatile metadata mode */
 	if (cache->metadata.is_volatile)
-		OCF_CMPL_RET(cache, priv, -EINVAL);
+		OCF_CMPL_RET(cache, priv, -OCF_ERR_INVAL);
+
+	/* Load is not allowed with 'force' flag on */
+	if (cfg->force) {
+		ocf_cache_log(cache, log_err, "Starting cache with 'load' "
+				"and 'force' options simultaneously is forbidden.");
```

Comment:
Actually from OCF point of view "load" is not an option - is operation that has it's own dedicated function. So I'd reword this message to something like "Using 'force' flag is forbidden for load operation".